app/backend/src/clj/teet/project/project_commands.clj

```diff
@@ -28,14 +30,22 @@
               :thk.project/custom-end-m]
           project-eid))
 
+(defn update-vektorio-project-name? [db project-id project-name]
+  (let [vektorio-enabled? (environment/feature-enabled? :vektorio)
+        vektorio-config (environment/config-value :vektorio)]
+      (if vektorio-enabled?
+            (vektorio/update-project-in-vektorio! db vektorio-config project-id project-name)
+            true)))
```

Comment:
Now you are asking `update-vektorio-project-name?` and returning a `true` when the vektorio is not enabled?
You should probably create a `maybe-update-vektorio-project-name` function and call it after the transaction for the project updating has happened.